dd-sdk-android-ktx/src/test/kotlin/com/datadog/android/ktx/sqlite/SqliteDatabaseExtTest.kt

```diff
@@ -169,7 +166,7 @@ class SqliteDatabaseExtTest {
     @Test
     fun `M close the Span around transaction W transactionTraced() throws exception`() {
         // GIVEN
-        var caughtException: Throwable? = null
+        val caughtException: Throwable?
```

Comment:
we can also use `assertThrows`, maybe, for such tests?

Reply:
I am not sure `assertThrows` allows to assert by reference. It only asserts that a specific exception type was thrown. But I think I can use something like: `assertThat(catchThrowable(...)).isEqualTo(fakeException)`

Reply:
`assertThrows` can return a value, so it can be:

```
val caughtException = assertThrows<Throwable> {
}
assertThat(caughtThrowable).isEqualTo(fakeException)
```

Reply:
yup kind of like that. Check my fixup.